src/routes/resources/+page.svelte

```diff
@@ -4,7 +4,34 @@
   export let data: PageData;
 
   let resources = data.payload.resources;
-  let displayedResources = resources;
+  var beginCount = 0;
+  var lastElem = 20;
+  const numOFDisplayElements = 5;
```

Comment:
In `handelPageBack` and `handelPageForward`, `numOFDisplayElements ` is added to the start and end. Proper pagination would only work if `lastElem = numOFDisplayElements` at the start, but this instead creates a sliding window effect where "Next page" loads in 5 extra resources and unloads the 5 at the start.

Reply:
Problem solved.